### DISCUSSION TITLE
Documentation on scale-out techniques

### DISCUSSION THREAD

**Discussion by nsmith-** [scikit-hep/coffea#731] [Ideas]
This discussion serves to collect links to existing documentation for scaling coffea analysis on multi-machine clusters.

Some first examples:
- WorkQueue + HTCondor https://coffeateam.github.io/coffea/wq.html#intro-coffea-wq
- Dask + HTCondor@LPC https://github.com/CoffeaTeam/lpcjobqueue
- Dask + HTCondor@CERN https://batchdocs.web.cern.ch/specialpayload/dask.html
- Coffea-casa (dask+HTCondor@UNL) https://coffea-casa.readthedocs.io/en/latest/

As can already be seen from the user interfaces of the above examples, the boundary between what software infrastructure is the responsibility of the analysis facility and what is the responsibility of the user varies across facilities. Hopefully this documentation can be distilled into a set of working examples for each site that can confirm that coffea is functional at scale with the site resources, and perhaps pave the way towards a more consistent API boundary.

**Comment by andrzejnovak** [comment]
Here's an older quick take on what the collection might look like https://github.com/andrzejnovak/coffea_configs 
The idea was one could clone the repo and try running a dummy processor with the stored configs, seeing which ones work at the local site.
